src/plone/restapi/services/discussion/conversation.py

```diff
@@ -75,7 +75,7 @@ def reply(self):
                          plone.protect.interfaces.IDisableCSRFProtection)
 
         conversation = IConversation(self.context)
-        if self.comment_id and self.comment_id not in conversation.keys():
+        if self.comment_id and self.comment_id not in list(conversation.keys()):
```

Comment:
this should be written: `if self.comment_id and self.comment_id not in list(conversation):`